### PR TITLE
Revert bidirectional & search

### PR DESCRIPTION
**Pull request by TGazica**
PRs being reverted:
- https://github.com/GetStream/stream-chat-android/pull/3818
- https://github.com/GetStream/stream-chat-android/pull/3861

### 🎯 Goal

Revert bidirectional pagination for release.

#### General
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `release` branch
- [ ] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_

### CODE REVIEW

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt [hunk 1]
```diff
@@ -105,16 +84,7 @@ public fun Messages(
     onScrolledToBottom: () -> Unit,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(vertical = 16.dp),
-    helperContent: @Composable BoxScope.() -> Unit = {
-        val coroutineScope = rememberCoroutineScope()
-        DefaultMessagesHelperContent(messagesState, lazyListState) {
-            coroutineScope.launch {
-                coroutineScope.launch {
```
Comment (MarinTolic) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt:112]: Was launching a coroutine in a coroutine here necessary or was it just a small mistake?

Comment (TGazica) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt:112]: mistake, no reason to launch a coroutine inside a coroutine

File: stream-chat-android-state/.env [hunk 2]
```diff
@@ -1,3 +0,0 @@
-STREAM_LOG_LEVEL=ALL
```
Comment (bychkovdmitry) [inline stream-chat-android-state/.env:1]: Was this part of this PR?

File: stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt [hunk 3]
```diff
@@ -122,7 +121,11 @@ internal class ChannelLogic(
     override suspend fun onQueryChannelRequest(channelType: String, channelId: String, request: QueryChannelRequest) {
         channelStateLogic.refreshMuteState()
 
-        runChannelQueryOffline(request)
+        /* It is not possible to guarantee that the next page of newer messages is the same of backend,
+         * so we force the backend usage */
+        if (!request.isFilteringNewerMessages()) {
```
Comment (bychkovdmitry) [inline stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt:126]: Also, was not part of the PR

Comment (kanat) [reply stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt:126]: It seems like this PR reverts both changes
